packages/env/lib/commands/start.js

```diff
@@ -140,6 +140,27 @@ module.exports = async function start( { spinner, debug, update, xdebug } ) {
 			: [],
 	} );
 
+	await dockerCompose.exec( 'wordpress', 'chown www-data:www-data wp-config.php', {
+			config: dockerComposeConfigPath,
+			log: debug,
+	} )
+
+	await dockerCompose.exec( 'wordpress', 'chmod 777 wp-config.php', {
+			config: dockerComposeConfigPath,
+			log: debug,
+	} )
+
+	await dockerCompose.exec( 'tests-wordpress', 'chown www-data:www-data wp-config.php', {
+			config: dockerComposeConfigPath,
+			log: debug,
+	} )
+
+	await dockerCompose.exec( 'tests-wordpress', 'chmod 777 wp-config.php', {
+			config: dockerComposeConfigPath,
+			log: debug,
+	} )
+
+
```

Comment:
I'm not sure that manually changing permissions for `wp-config.php` is the right fix, see discussion at https://github.com/WordPress/gutenberg/pull/29770 :sweat_smile:

Reply:
I think you have a good point that configuration should be set idiomatically via docker-compose configuration.

However, I think it would still make sense to fix `wp-config.php` permissions, because currently `wp-config.php` is owned by root and so any commands that need to use the config for any reason would fail.

Reply:
We've had to update permissions for some of these files to get them to work in wp-env before, so I'm not against it

Reply:
I don't think we should use `WORDPRESS_CONFIG_EXTRA` because the `wordpress` image will only use that variable  if `wp-config.php` doesn't already exist which isn't very flexible for developers.

Reply:
I don't think we should use `WORDPRESS_CONFIG_EXTRA` because the `wordpress` image will only use that variable  if `wp-config.php` doesn't already exist which isn't very flexible for developers. For example: you wouldn't be able to update `WP_DEBUG` in `.wp-env.json`, restart `wp-env`, and see WordPress running in debug mode.

Reply:
> I don't think we should use `WORDPRESS_CONFIG_EXTRA` because the `wordpress` image will only use that variable if `wp-config.php` doesn't already exist

I'm not totally sure, but I _think_ that's no longer true per https://github.com/docker-library/wordpress/pull/557, which has `wp-config.php` read those values dynamically from the docker env (using a `getenv()` helper); see e.g. 

- `WORDPRESS_DEBUG`: https://github.com/docker-library/wordpress/pull/557/files#diff-7f28a0f62019f2d8b061638e8c3aea207404c1499f1c9241db3216fadd8f4530R99 or
- `WORDPRESS_CONFIG_EXTRA`: https://github.com/docker-library/wordpress/pull/557/files#diff-24f50e34cd95d8171662daf812269748e400594db64a7ee63ffd1fb3db5d9965R108-R110